Gunma animation u

DueyDu opened the pull request:
### Background:

Update main menu to have a nice clean animated screen as well as update Gunma-Chan takes a Hike animations so it is more fluid and responsive to gameplay.

### Change:

Updated MainMenu.java and GameScreen.java to be updated with new and improved animations.

## Tests:

Made sure animation runs smoothly and every option has been checked for crashes.

DueyDu committed:
updated game animations for gunma and frenemy

DueyDu committed:
New Assets Added

DueyDu committed:
Updated Sounds Files from WAV to MP3

DueyDu committed:
Fixed to match with Master

DueyDu committed:
Added Animated Main Menu with Characters

sjodon commented on core/src/asu/gunma/ui/screen/game/GameScreen.java:
```diff
@@ -127,16 +144,15 @@
     Preferences prefs;
 
-    public GameScreen(Game game, ActionResolver speechGDX, Music music, DbInterface dbCallback, Screen previous, ArrayList<VocabWord> activeList, Preferences prefs, GameAssets gameAssets) {
+    public GameScreen(Game game, ActionResolver speechGDX, Music music, DbInterface dbCallback, Screen previous, ArrayList<VocabWord> activeList, Preferences prefs) {
```
I think this is just from the new branch overwriting the addition of `GameAssets`, but the build fails without this.
```suggestion
    public GameScreen(Game game, ActionResolver speechGDX, Music music, DbInterface dbCallback, Screen previous, ArrayList<VocabWord> activeList, Preferences prefs, GameAssets gameAssets) {
```

sjodon commented on core/src/asu/gunma/ui/screen/game/GameScreen.java:
```diff
@@ -127,16 +144,15 @@
         this.activeVList = activeList;
-        this.gameAssets = gameAssets;
-        gameMusic = Gdx.audio.newMusic(Gdx.files.internal(gameAssets.introMusicPath));
+        gameMusic = Gdx.audio.newMusic(Gdx.files.internal("IntroMusic.mp3"));
```
same reason as above
```suggestion
        this.gameAssets = gameAssets;
        gameMusic = Gdx.audio.newMusic(gameAssets.introMusicPath);
```

sjodon reviewed (changes requested):
I tried it out and it looks awesome!!! The build was failing though because I think the `GlobalAsset` changes were overwritten by your changes in `GameScreen.java` and `MainMenuScreen.java` I pointed out a couple spots here, but there were a few more I think. Once those were fixed it worked great!

sydneyjodon committed:
Add score screen to the end of each level

DueyDu committed:
Adapted Gunma File to use Global Game Assets

Utilizes global game assets instead of simply just putting in name of animation, will come in handy for implementation of firebase

DueyDu committed:
Updated Game Assets

Now Gunma Chan is smaller when holding his bag, sprite was created by Sydney

DueyDu committed:
Main Menu Updated

Made to resolve commit errors

DueyDu committed:
Game Screen Bug Fix

Fixed for commit error

DueyDu committed:
Game Assets Fix

Fix Game Assets

DueyDu committed:
Added Localization Stuff

DueyDu committed:
Merge branch 'master' into Gunma_animation_u

# Conflicts:
#	core/src/asu/gunma/ui/screen/game/GameScreen.java
#	core/src/asu/gunma/ui/screen/menu/MainMenuScreen.java
#	core/src/asu/gunma/ui/util/AssetManagement/GameAssets.java
